sandbox/feature-toggle-example/src/main/java/com/sourcefuse/jarc/sandbox/featuretoggleexample/handlers/HelloHandler.java

```diff
@@ -0,0 +1,16 @@
+package com.sourcefuse.jarc.sandbox.featuretoggleexample.handlers;
+
+import com.sourcefuse.jarc.services.featuretoggleservice.annotation.FeatureHandlers;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Slf4j
+@Component("HelloHandler")
```

Comment:
can we change name of this handler to TenantHandler or an meaningful name

Reply:
Removed were not being used